packages/php-storage-driver-common/proto/info.proto

```diff
@@ -47,10 +47,33 @@ message DatabaseInfo {
 message SchemaInfo {
   repeated ObjectInfo objects = 1;
 }
+
 /**
- * TODO
- */
-message ViewInfo {}
+   * Table column definition
+   */
+message Column {
+  string name = 1;
+  string type = 2;
+  string length = 3;
+  bool nullable = 4;
+  string default = 5;
+  google.protobuf.Any meta = 6; // metadata specific for each backend
+  /**
+   * Teradata table column metadata
+   */
+  message TeradataTableColumnMeta {
+    bool isLatin = 1;
+  }
+}
```

Comment:
Tady to nedělaj bude to zbytečně BC, protože tak jak to používáme sa ti změní namespace a budeš muset udělat update TD driveru třeba, normálně to udělaj jako TableInfo.Column